libs/wingsdk/test/core/plugin-manager.test.ts

```diff
@@ -196,3 +198,94 @@ test("plugins are run in order they are passed in", () => {
   // THEN
   expect(tfResourcesOfCount(postSynthOutput, "aws_s3_bucket")).toEqual(0);
 });
+
+describe("test plugin examples", () => {
+  const EXAMPLES_DIR = "../../examples/plugins";
```

Comment:
FYI @hasanaburayyan for the future, I think we should host these examples into the SDK directory, or move the tests to elsewhere (like the hangar). 

One reason is puristic  -- I think it will be easier to maintain the SDK long term when it's self contained, so that none of source code or test code depends on other libraries in the monorepo.

But the practical reason is that by referencing the examples directory, we are introducing a dependency that NX (our task runner) isn't aware of. So if we want to keep the tests here / reference examples from outside, we should update our NX configuration files -- otherwise it's possible for someone to ask NX to run the SDK tests, update an example, and then when they ask NX to run the SDK tests again, it will think no files have changed (because the examples directory wasn't mentioned as a dependency) and it will just return the cached result.

Reply:
@Chriscbr thanks for the catch and great points. I’m going to get these moved (just been procrastinating)